packages/website/plugins/generated-rule-docs.ts

```diff
@@ -193,35 +194,79 @@ export const generatedRuleDocs: Plugin = () => {
         type: 'paragraph',
       } as mdast.Paragraph);
 
+      // need this, so we don't have comments inside the JSON objects
+      const getEslintrcStr = (withComment: boolean): string => {
```

Comment:
[Nit] A couple of small changes:

* I'm not a fan of abbreviations like "Str", as they can be hard to read - especially for ESL (English Second Language) folks.
* A `/** ... */` JSDoc would be more standard to get intellisense matching up the comment to the function

```suggestion
      /**
       * @param withComment Whether to include a full comment note.
       * @remarks `withComment` can't be usedd inside a JSON object.
       */
      const getEslintrcStr = (withComment: boolean): string => {
```

How do you feel about this?